quartz_solar_forecast/data.py

```diff
@@ -111,20 +114,45 @@ def format_nwp_data(df: pd.DataFrame, nwp_source:str, site: PVSite):
     )
     return data_xr
 
+def get_enphase_data(enphase_user_id: str, enphase_api_key: str) -> float:
```

Comment:
could this be moved to a seperate folder, say `quartz_solar_forecast/inverters/enphase.py`

Reply:
> could this be moved to a seperate folder, say `quartz_solar_forecast/inverters/enphase.py`

Added a /inverters/enphase.py file with the required changes in #66